### PR TITLE
Branch 24.06 RAPIDS 24.06

### PR DESCRIPTION
**Pull request by cwharris**
## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/nv-morpheus/Morpheus/blob/main/docs/source/developer_guide/contributing.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.

### CODE REVIEW

File: cmake/morpheus_utils/package_config/matx/patches/matx_cccl_interface_fix.patch [hunk 1]
```diff
@@ -1,22 +1,18 @@
+From 6ddba385b27f4f7075fcdd5a0916f59944c3ec16 Mon Sep 17 00:00:00 2001
+From: Christopher Harris <xixonia@gmail.com>
```
Comment (mdemoret-nv) [inline cmake/morpheus_utils/package_config/matx/patches/matx_cccl_interface_fix.patch:2]: Dont want your personal email in here

**Review by mdemoret-nv — APPROVED** [review]

File: cmake/morpheus_utils/package_config/matx/patches/matx_cccl_interface_fix.patch (hunk 1 above)
Comment (cwharris) [reply cmake/morpheus_utils/package_config/matx/patches/matx_cccl_interface_fix.patch]: no I do not. :)

**Pull request merged**